tests/ltp/README.md

```diff
@@ -1052,7 +1052,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/sendmsg/sendmsg01 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/sendmsg/sendmsg02 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/sendmsg/sendmsg03 | 0 | 0 | 0 |
-| /ltp/testcases/kernel/syscalls/sendto/sendto01 | 1 | 1 | 1 |
+| /ltp/testcases/kernel/syscalls/sendto/sendto01 | 0 | 0 | 0 |
```

Comment:
Did sendto01 test pass in the past?

Reply:
Same as bind03, there was another PR to remove it